script/build.js

```diff
@@ -652,7 +655,7 @@ if (options.buildtype !== 'development') {
     Object.keys(files).forEach((filename) => {
       if (filename.match(/\.(html|css)$/) !== null) {
         Object.keys(manifest).forEach((originalAssetFilename) => {
-          const newAssetFilename = manifest[originalAssetFilename];
+          const newAssetFilename = manifest[originalAssetFilename].replace('/generated/', '');
```

Comment:
I wasn't ever super sure of what the deal was with `generated/`--why take it out of the file (path) name?

Reply:
It needs to be removed to match another name in the manifest (I think, I've already forgotten part of this, I need to write a comment explaining it)